nservicebus/azure-service-bus/dlq-forwarding.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Azure Service Bus DLQ Forwarding
+versions: '[7,)'
+reviewed: 2016-08-29
+tags:
+- Azure
+- Cloud
+- Error Handling
+---
+
+## Dead Letter Queue Forwarding
```

Comment:
@SimonCropp The rendered title in the menu will be "DLQ Forwarding", right? In that case perhaps "Azure Service Bus Dead Letter Queue Forwarding" is a better option @yvesgoeleven ?